packages/datadog-plugin-mysql/test/index.spec.js

```diff
@@ -111,23 +105,22 @@ describe('Plugin', () => {
             done()
           })
 
-          connection.query('SELECT 1 + 1 AS solution')
+          connection.query('SELECT 1 + 1 AS solution', done())
```

Comment:
This seems wrong. It looks like you're calling the `done` before passing it it as the second parameter to `query`. You probably want to just pass `done` without calling it, since `query` expects a callback, not the return value of a callback.

Reply:
Actually, reading the surrounding code, and the test name, you definitely want to keep this line as it was originally.